requirements.txt

```diff
@@ -2,10 +2,11 @@
 colour-runner==0.0.4
 coverage==4.2
 dj-database-url==0.4.1
-django>=1.8,<1.9
+django>=1.10,<1.11
```

Comment:
`pgcrypto` is required as well. 
I can see the issue with including files. I assumed there was some way to specify a requirement to be met which isn't force installed. I've had a look at the docs and can't see a way to do this.
Today I learned.